test/features/edgeconnect/edgeconnect.go

```diff
@@ -49,13 +48,16 @@ func NormalModeFeature(t *testing.T) features.Feature {
 
 	edgeConnectTenantConfig := &tenantConfig{}
 
-	builder.Assess("create EC configuration on the tenant", createTenantConfig(secretConfig, edgeConnectTenantConfig))
+	testECname := fmt.Sprintf("test-edgeconnect-normal-%d", rand.IntN(100))             //nolint
+	testHostPattern := fmt.Sprintf("e2eTestHostPattern%d.internal.org", rand.IntN(100)) //nolint
```

Comment:
could we avoid putting the `//nolint` thing everywhere? why have linters if we just turn them off?

I know that they are "just tests" but still

also is rand of 100 random enough? we used to have similar logic for the network-zone test, and it was still flaky
- couldn't we use `github.com/google/uuid` ? or would that be overkill? (we already have it as a indirect dependency )

Reply:
sure, i can try it.

Reply:
@0sewa0 fixed in 131b746a